python/hail/methods/misc.py

```diff
@@ -123,16 +124,23 @@ def maximal_independent_set(i, j, tie_breaker=None):
                                                          j._ast.to_hql(),
                                                          joption(tie_breaker_hql)))
 
+
+@handle_py4j
+def require_variant(dataset, method):
+    if (dataset.row_key != ['locus', 'alleles'] or
```

Comment:
Do we want to require the names?  I'm OK with this, but I thought we should make sure we ask the question.

Reply:
I think it's a good idea. 

We also need to add `rename` to MatrixTable, thinking of this.

Reply:
to clarify, I'm happy to hear alternate perspectives!